### PR TITLE
Set the latest result as default results widget

### PR DESCRIPTION
**Pull request by gregori0o**

### CODE REVIEW

File: src/widgets/unfold_widgets/results_widget.py [hunk 1]
```diff
@@ -23,10 +23,18 @@ def load_widget(self):
...
+                result_idx = 0
                 for i, result_widget in enumerate(results):
-                    algorithm_result_tab_widget.addTab(result_widget, f'{i+1}')
-                self.results_tab_widget.addTab(algorithm_result_tab_widget, f'{technique}: {algorithm}')
-
+                    result_idx = algorithm_result_tab_widget.addTab(result_widget, f'{i+1}')
+                algorithm_idx = self.results_tab_widget.addTab(algorithm_result_tab_widget, f'{technique}: {algorithm}')
+                if technique == last_technique and algorithm == last_algorithm:
+                    self.results_tab_widget.setCurrentIndex(algorithm_idx)
+                    self.results_tab_widget.widget(algorithm_idx).setCurrentIndex(result_idx)
```
Comment (mhawryluk) [inline src/widgets/unfold_widgets/results_widget.py]: do we need the `result_idx` variable? we could set the index here to `len(results) - 1`

File: src/widgets/unfold_widgets/results_widget.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from PyQt5.QtWidgets import QHBoxLayout, QTabWidget
+from PyQt5.QtWidgets import QHBoxLayout, QTabWidget, QWidget
```
Comment (mhawryluk) [inline src/widgets/unfold_widgets/results_widget.py]: `QWidget` is unused

File: src/widgets/unfold_widgets/results_widget.py (hunk 1 above)
Comment (mhawryluk) [reply src/widgets/unfold_widgets/results_widget.py]: assuming the `addTab` method returns indices counting from 0

**Review by mhawryluk — APPROVED** [review]
🎉

**Review by t0maboro — APPROVED** [review]
👍

**Pull request merged**